<ui version="4.0" >
 <class>BookmarkManager</class>
 <widget class="QDialog" name="BookmarkManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Bookmark Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Actions</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QPushButton" name="button_edit" >
        <property name="text" >
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_delete" >
        <property name="text" >
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_revert" >
        <property name="text" >
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_export" >
        <property name="text" >
         <string>Export</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_import" >
        <property name="text" >
         <string>Import</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QTableView" name="bookmarks_table" >
     <property name="showDropIndicator" stdset="0" >
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="sortingEnabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BookmarkManager</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>225</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel" >
     <x>225</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BookmarkManager</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>225</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel" >
     <x>225</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
